ui/ts/typings/chai/chai.d.ts

```diff
@@ -0,0 +1,308 @@
+// Type definitions for chai 2.0.0
```

Comment:
can we use https://github.com/DefinitelyTyped/tsd for these rather than vendoring them? https://www.npmjs.com/package/tsd